test/Microsoft.Framework.Internal.Test/project.json

```diff
@@ -0,0 +1,24 @@
+﻿{
+    "compilationOptions": {
+        "warningsAsErrors": "true"
+    },
+    "dependencies": {
+        "Microsoft.Framework.CopyOnWriteDictionary.Internal": "1.0.0-*",
+        "Microsoft.Framework.NotNullAttribute.Internal": "1.0.0-*",
+        "Microsoft.Framework.PropertyHelper.Internal": "1.0.0-*",
+        "Microsoft.Framework.PropertyActivator.Internal": "1.0.0-*",
```

Comment:
Build time dependencies are equivalent to regular dependencies for test projects, don't care enough though, will make the change.